docs/topics/cache.txt

```diff
@@ -77,17 +77,15 @@ database or filesystem usage.
 
 After installing Memcached itself, you'll need to install a Memcached
 binding. There are several Python Memcached bindings available; the
-three most common are `python-memcached`_, `pylibmc`_, and `pymemcache`_.
+two most common are `pylibmc`_ and `pymemcache`_.
```

Comment:
I think we don't want to say "most common" anymore as `python-memcached` is probably widely used. So I'd go for something like this:
```suggestion
two supported by Django are `pymemcache`_ and `pylibmc`_.
```